src/pages/components/checkbox/usage.mdx

```diff
@@ -1,33 +1,48 @@
 ---
 title: Checkbox
-description: Checkboxes are used for a list of options where the user may select multiple options, including all or none.
+description: Checkboxes are used when there are one or more items to select in a list.
 tabs: ["Code", "Usage", "Style", "Accessibility"]
 ---
 
+<PageDescription>
+
+Checkboxes are used when there are one or more items to select in a list.
+
+</PageDescription>
+
 <AnchorLinks>
 
-<AnchorLink>General guidance</AnchorLink>
-<AnchorLink>Related components</AnchorLink>
+<AnchorLink>Overview</AnchorLink>
+<AnchorLink>Formatting</AnchorLink>
+<AnchorLink>Behaviors</AnchorLink>
 <AnchorLink>Content</AnchorLink>
-<AnchorLink>Tables</AnchorLink>
+<AnchorLink>Related</AnchorLink>
+<AnchorLink>Feedback</AnchorLink>
+<AnchorLink>References</AnchorLink>
 
 </AnchorLinks>
 
-## General guidance
+## Overview
 
-Checkboxes are used for a list of options where the user may select multiple options, including all or none.
+Checkboxes are used when there are one or more items to select in a list. Each checkbox works independently from other checkboxes in the list, therefore checking an additional box does not affect any other selections.
 
-#### Checkbox states
+### When to use
 
-The checkbox control allows for three states: **selected**, **unselected**, and **indeterminate**. The indeterminate state comes into play when the checkbox contains a sublist of selections, some of which are selected, and some unselected.
+#### Forms
 
-#### Click target
+Can be used in forms on a full page, in modals, or on side panels.
+
+#### Filtering and batch action
+
+Used to filter data either on a page, in a menu, or within a component. Checkboxes are found in our [Multiselect dropdown](components/dropdown/code) to select multiple items and in [Data table](components/data-table/code) for batch editing purposes.
```

Comment:
Let me know if this would be confusing or not, but on the development side a `Checkbox` component would correspond to this definition from WAI ARIA:

> Dual-state: The most common type of checkbox, it allows the user to toggle between two choices -- checked and not checked.

For selection in a component, like for multi-select, they are styled to show that an option is selected but unfortunately we don't use `Checkbox` in this situation.

The difference is that a checkbox corresponds to the following input in HTML:

```html
<label for="checkbox">This is a checkbox</label>
<input id="checkbox" type="checkbox">
```

Since `input`'s are interactive, they unfortunately aren't able to be used in things like multi-select.

Reply:
so this is not a checkbox situation for the multiselect dropdown. is that also true for the selectable rows in a data table?

Reply:
I believe for rows in data table we still use a checkbox! This is definitely controversial but seems fair to have it in here since there is no official solution for row selection in a table right now so really nothing works 🤦 lol